integration/pom.xml

```diff
@@ -806,6 +812,16 @@
 <!--                <module>dataservice-hosting-tests/tests-integration/tests</module>-->
             </modules>
         </profile>
+        <profile>
+            <id>full</id>
+            <modules>
+                <module>samples/product</module>
+                <module>automation-extensions</module>
+                <module>tests-common</module>
+                <module>mediation-tests</module>
+                <module>cli-tests</module>
```

Comment:
Let's add the commented out as well <module>dataservice-hosting-tests</module>

Reply:
Done